ml-agents/mlagents/trainers/meta_curriculum.py

```diff
@@ -32,9 +32,9 @@ def __init__(self, curriculum_folder: str):
         try:
             for curriculum_filename in os.listdir(curriculum_folder):
                 # This process requires JSON files
-                if not curriculum_filename.lower().endswith(".json"):
+                brain_name, extension = os.path.splitext(curriculum_filename)
```

Comment:
So far I don't think this has tripped anybody up. But the old behavior would mean that a brain with a `.` in the name wouldn't be able to match up with the metacurriculum, because `curriculum_filename.split(".")[0]` would take the only the filename up to the first `.` instead of the last one.